changelog/20250909_feature_improved_agent_certificate_rotation.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Improved agent certificate rotation
+kind: feature
+date: 2025-09-09
+---
+
+* Database Pods now mount a secret containing both old and new certificates, with file names being the hash of the certificate. When it is time to rotate the certificate, the operator updates the automation config with a new path (including the hash) to the certificate. This eliminates the need to manually restart Pods during certificate rotation.
```

Comment:
Release notes are too technical imo. "How" - we have in PR's description. Here we should just say "what". 
For example: "Improved agent certificate rotation. Pods are automatically restarted now when the agent's certificate is rotated."

Reply:
Great feedback, thanks! I'm waiting for more feedback to come in and then I'll update the release note together with other changes.